news/_posts/2023-07-31-playtest-20230731.md

```diff
@@ -0,0 +1,32 @@
+---
+kind: article
+title: "Playtest 20230731"
+author: "PunkPun"
+created_at: 2023-07-31 22:00 +0000
+disqus_id: "playtest-20230731"
+permalink: "/news/playtest-20230731/"
+---
+
+As summer is in full swing, we are excited to kick off the playtest series for the upcoming OpenRA release! This time around, it is all about spring cleaning: squashing bugs, implementing minor requested features, and preparing for larger reworks. But fear not! We still have a bunch of cool stuff lined up just for you! Without further ado, here are some notable changes you can expect in [Playtest 20230731]({{ '/download' | relative_url }}):
+
+* Added new Covert Ops missions for Tiberian Dawn: "Eviction Notice" and "Twist of Fate"
+* Added new Red Alert missions "Allies10b" and "Soviet13a" and the Counter-Strike mission "Mousetrap"
+* Fixed several palette remapping issues and restored ability to select darker player colors
+* Implemented a new custom Lua wrapper, improving error reporting for map scripters
+* Introduced a new modular asset installer
+* Reworked sprite sequences significantly improving modding support and asset installation
+
+<figure>
+  <video autoplay loop muted>
+    <source src="{{ '/images/news/20230731-smoke.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20230731-smoke.webm' | relative_url }}" type="video/webm">
+    <img src="{{ '/images/news/20230731-smoke.png' | relative_url }}"/>
+  </video>
+  <figcaption>One of the smaller features is a new smoke emitter for D2K's damaged vehicles which better matches the original</figcaption>
+</figure>
+
+For more information see the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/).
+
+And before we wrap up, we would like mention that this release contains significant progress towards C&C Remastered Collection support, including the installation of art assets from the new EA App. "Tiberian Dawn HD" playtest is a separate release and can be [downloaded here](https://github.com/OpenRA/TiberianDawnHD/releases/tag/release-20230731). This preview is multiplayer-compatible with the main 20230731 playtest. However, please keep in mind that performance, memory usage and loading times have not been optimized yet. The C&C Remastered Collection must be installed through Steam or the EA App, and incase you are using macOS or Linux the project README provides you [detailed installation instructions.](https://github.com/OpenRA/TiberianDawnHD#asset-installation)
```

Comment:
```suggestion
And before we wrap up, we would like mention that this release contains significant progress towards C&C Remastered Collection support, including the installation of art assets from the new EA App. The "Tiberian Dawn HD" playtest is a separate release and can be [downloaded here](https://github.com/OpenRA/TiberianDawnHD/releases/tag/release-20230731). This preview is multiplayer-compatible with the main 20230731 playtest. However, please keep in mind that performance, memory usage and loading times have not been optimized yet. The C&C Remastered Collection must be installed through Steam or the EA App, and in case you are using macOS or Linux the project README provides you [detailed installation instructions](https://github.com/OpenRA/TiberianDawnHD#asset-installation).
```